cmake/Modules/LocalLibyuv.cmake

```diff
@@ -34,6 +34,7 @@ else()
         GIT_REPOSITORY "https://chromium.googlesource.com/libyuv/libyuv"
         BINARY_DIR "${LIBYUV_BINARY_DIR}"
         GIT_TAG "${AVIF_LOCAL_LIBYUV_TAG}"
+        PATCH_COMMAND sed -i.bak -e "s:find_package.*(.*JPEG.*)::" CMakeLists.txt
```

Comment:
Vincent: Are you planning to submit a patch to libyuv to handle this properly?

Looking at the other build systems in the libyuv source tree, I think we should add a LIBYUV_DISABLE_JPEG option to libyuv's CMakeLists.txt file.

Reply:
No need actually, I reverted that patch: there is actually an official way to make find_package fail, cf https://github.com/AOMediaCodec/libavif/pull/2473